Jenkinsfile

```diff
@@ -54,12 +54,15 @@ node {
 
     // Deploy
     stage 'Deploy'
-    sh "cat marathon.json | jq \".container.docker.image = \\\"mesosphere/cd-demo-app:${gitCommit()}\\\" | .id = \\\"jenkins-deployed-app\\\" | .labels.lastChangedBy = \\\"${gitEmail()}\\\"\" > marathon-rendered.json"
+
     marathon(
         url: 'http://marathon.mesos:8080',
         forceUpdate: false,
         credentialsId: 'dcos-token',
```

Comment:
Maybe for another ticket. It is probably worth documenting how to do this demo manually and this would need to be documented to support that.